src/lib/formats/micropolis_hd.cpp

```diff
@@ -0,0 +1,103 @@
+// license:BSD-3-Clause
+// copyright-holders:Eric Anderson
+/***************************************************************************
+
+Micropolis MFM HDD format
+
+See vgi_dsk.cpp for the related floppy format. The sector format is:
+SYNC (0xFF), head, track, sector, 256 byte data, 4 byte ECC.
+
+****************************************************************************/
+
+#include "micropolis_hd.h"
+
+static const int sectors = 32;
+static const int sector_size = 256;
```

Comment:
Please use `constexpr` when appropriate, and use `SCREAMING_SNAKE_CASE` for constants.